packages/plugins/Tips/src/hooks/useProfilePublicKey.ts

```diff
@@ -7,10 +7,9 @@ export function useProfilePublicKey(userId?: string) {
     const { networkIdentifier } = useBaseUIRuntime()
     const platform = resolveNetworkToNextIDPlatform(networkIdentifier)
     return useQuery({
-        enabled: !!userId && !!platform,
         queryKey: ['next-id', 'lasted-active', platform, userId],
         queryFn: async () => {
-            if (!userId || !platform) return
+            if (!userId || !platform) return null
```

Comment:
Why? It seems to cause unnecessary changes.

Reply:
This is not absolutely necessary